site/source/docs/api_reference/preamble.js.rst

```diff
@@ -234,7 +234,7 @@ Conversion functions — strings, pointers and arrays
 
 .. js:function:: intArrayFromString(stringy, dontAddNull[, length])
 
-  This converts a JavaScript string into a C-line array of numbers, 0-terminated.
```

Comment:
What does `C-line` mean?

Reply:
It's either a typo for `C-like` or it's a variant term I've not heard. Shall we change it while in there?

Reply:
Probably meant to be `C-like`, yeah. I think it's worth fixing.

Also maybe worth adding to the Changelog that `intArrayToString` now supports UTF8 input, as this PR does change the API (but by supporting stuff that broke before, so not a risky change).